cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -33,7 +34,15 @@ func NewCmd() *cobra.Command {
 		Long:   `Has flags to configure dsl, bundle, and selector.`,
 		Hidden: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Info("TODO")
+			scorecardFlags := scorecard.ScorecardFlags{
```

Comment:
You can move this to the top of `NewCmd()` and use its fields as flag variables instead of those pesky globals.

Reply:
And let's not call it `ScorecardFlags` because it stutters, and the scorecard library shouldn't be concerned with flags. How about simple `Options`?

Reply:
changed in latest commit.